docs/content/try-out.md

```diff
@@ -8,33 +8,34 @@ menu: shortcuts
 
 To quickly see Athens in action, follow these steps:
 
-First, make sure you have [Go 1.11 installed](https://gophersource.com/setup/) and that GOPATH/bin is on your path.
+First, make sure you have [Go 1.11 installed](https://gophersource.com/setup/),
+that GOPATH/bin is on your path, and that you have enabled the [Go
+Modules](https://github.com/golang/go/wiki/Modules) feature by exporting
+GO111MODULE=on.
```

Comment:
what do you think about showing the commands for bash and powershell on how to export `GO111MODULE=on`?

that'd be important in case someone does check out the source into their GOPATH

Reply:
Was considering whether it made the page too busy. That said in retrospect putting it at the end of a long sentence it seems less prominent than it should be (given how new modules are). I'll update it now with your suggestion!